idaes/ui/flowsheet.py

```diff
@@ -280,6 +281,7 @@ def _add_unit_model_with_ports(self, unit):
                 stream_df = stream_df.reset_index().rename(
                     columns={"index": "Variable"}
                 )
+                stream_df = stream_df.replace(np.nan, "NaN", regex=True)
```

Comment:
What's the reason to use `regex=True`? As I understand from the docs, this will cause the first argument to be interpreted as a regex pattern, which doesn't seem to be applicable is the argument is `np.nan`

Reply:
I think that was just a copy from stackoverflow. I removed it.